src/frameworks/index.ts

```diff
@@ -259,12 +259,22 @@ export async function prepareFrameworks(
       `\n${frameworksCallToAction(SupportLevelWarnings[support](name), docsUrl, "   ")}\n`
     );
 
+    const command = options._name;
+    if (typeof command !== "string") throw new Error("options._name is not a string");
```

Comment:
This is a pretty esoteric error to bubble up - we should make this a bit more user friendly. Even something like "options._name is not a string. This is likely a bug with firebase-tools - if you see this, please report an issue at https://github.com/firebase/firebase-tools/issues